doc/api/path.md

```diff
@@ -155,11 +223,15 @@ path.format({
 
 ## path.isAbsolute(path)
 
-Determines whether `path` is an absolute path. An absolute path will always
-resolve to the same location, regardless of the working directory. `path` must
-be a string.
+* `path` {String} A file path
```

Comment:
nit: this function isn't designed for just *file* paths. I'm not sure if it's better to find a better description than "A path" or to just remove it completely.